Milestone1.md

```diff
@@ -32,26 +32,26 @@
 
   - Again, this will be a scaler with 1-10 scale.
 
- > Questions haven't been used: Gender (still considering adding!), Education Major, how often they recycle, how often do they think about eating less meat, flying vs driving vs biking/walking, how do they commute to school, did their parents or culture hav sustainability-conscious values, have they studied the environment or taken any sustainability courses.
+ > Questions haven't been used: Gender, Education Major, how often do they think about eating less meat, how do they commute to school, have they studied the environment or taken any sustainability courses.
 
  ### Other question we aim to answer
 
- Additionally, we would like to know if the video about recycling *changes* their opinion on the importance of sustainability.
+ Additionally, we would like to know if the video about recycling *changes* their opinion on the importance of sustainability and if so which predictors plays role in this change.
 
  ### Plan for Analyzing the Survey Results
 
- **Null hypothesis**: A person’s frequency of recycling does not influence her/his opinion and attitudes towards the importance of sustainability.
+ **Null hypothesis**: A persons opinion and attitudes towards the importance of sustainability is not affected by frequency of recycling, age, and whether they grow up in environmentally conscious family or not.
```

Comment:
I think that we still need the 's here (_person's opinion_ instead of _persons opinion_)

Reply:
Thank you for pointing it out. I fixed the typo.